src/graphql/auth.rs

```diff
@@ -141,6 +157,14 @@ impl PolicyCheck {
             .await
     }
 
+    pub async fn check_super_admin(
+        &self,
+        token: Option<&Authorization<Bearer>>,
+    ) -> Result<(), AuthError> {
+        self.authorise(&self.admin, SuperAdminRequest::new(token)?)
```

Comment:
I think this will need a new field to specify the super_admin query. Will also propagate to the CLI/env reading

Reply:
Actually, looking at the [auth rules](https://github.com/DiamondLightSource/authz/blob/main/policy/diamond/policy/admin/admin.rego), I think we can make do without changing anything. If the `beamline` field in `AdminRequest` is made optional, it acts the same as `super_admin` would if no beamline is given.

Reply:
I have changed this.